packages/python-packages/apiview-copilot/evals/_runner.py

```diff
@@ -1,28 +1,19 @@
-import json
 import os
 import pathlib
 import sys
-from typing import Any, Set
+from typing import List
 
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
 
 from azure.ai.evaluation import evaluate
 from azure.identity import AzurePipelinesCredential
-from evals._custom import CustomAPIViewEvaluator
-from src._apiview_reviewer import ApiViewReview
 from src._settings import SettingsManager
 from tabulate import tabulate
 
-DEFAULT_NUM_RUNS: int = 1
-
-
-def _review_apiview(query: str, language: str):
-    """Internal global callable for evals framework."""
-    reviewer = ApiViewReview(target=query, language=language, base=None)
-    review = reviewer.run()
-    reviewer.close()
-    return {"actual": review.model_dump_json()}
+from _config_loader import load_workflow_config, WorkflowConfigError, get_evaluator_class
```

Comment:
When running from CLI it won't find the module unless we add

```suggestion
from evals._config_loader import load_workflow_config, WorkflowConfigError, get_evaluator_class
```